src/max.js

```diff
@@ -6,13 +6,23 @@
  * @returns {number} The largest number
  */
 function max(array) {
-  // This is your job. :)
-  // Don't use the built-in Math.max
+let maxSoFar = array[0];
+
+for (let element of array) {
+if (element > maxSoFar) {
+maxSoFar = element;
```

Comment:
Please indent. Use your editor. I don't want to have to remind you to do this kind of thing by this point, it just becomes exhausting and demoralizing for me.